.editorconfig

```diff
@@ -283,6 +321,102 @@ ij_java_wrap_comments = false
 ij_java_wrap_first_method_in_call_chain = false
 ij_java_wrap_long_lines = false
```

Comment:
Grote verbetering Tim, top! 
Ik heb een paar vragen/opmerkingen, welke ik liever hier neerzet dan generiek in de PR, omdat die niet threaded werkt. 

1. In file `aws/src/main/java/org/frankframework/aws/AwsUtil.java` zie ik dat `// S3` comment wordt losgeknipt van de Javadoc.. Dat effect vind ik niet mooi. Geen idee of dat komt door deze wijzigen overigens. 
2. Wordt de XML-style guide nu niet ook toegepast op de Larva-XMLs en alles in de `resources` mappen? Of zit er een filter/scope op zodat het alleen in de  `java` mappen werkt?
3. Ik zie dat de keys in de file zijn geprefixt met `ij_`. Ik vraag me daarom ook af of het werkt in Eclipse. Heb vanmorgen gevraagd of Niels jouw branch wil uitchecken om het te verifieren. 
4. Sommige regels gaat hij nu over meerdere regels uitspreiden als het langer is dan `max_line_length` welke nu op 150 staat. Mij betreft mag die nog wel iets hoger (+10 oid), maar goed, ik gebruik ook high-res schermen, dus weet niet of dat ook jullie voorkeur heeft.

Reply:
@jkosternl ,

Bedankt voor de feedback!
1. Ik zal eens kijken of ik wat kan doen aan de single-line comments aan het einde van code-regels, of die kunnen blijven waar ze nu zijn
2. Ik heb bij het reformatten gekozen alleen Java files te reformatten, dus geen JSON YAML XML of property files/
3. Heel veel dingen zijn inderdaad IntelliJ-specifiek. Het zij zo, helaas. Ik zal ook een IntelliJ codestyle xml bestand toevoegen en kijken of er een plugin is waarmee de formatting rules ook naar Eclipse kunnen worden geexporteerd.
4. Ik kan de max line length wel wat ophogen maar sommige dingen zijn gewoon leesbaarder als je meerdere dingen onder elkaar zet, niet vanwege de breedte van je scherm maar gewoon omdat het overzichtelijker is

Ik denk er trouwens over om voor import-order nog wat regels toe te voegen om alle JUnit, Mockito etc imports of bovenaan, of onderaan te zetten. Zodat je in je imports alles bij elkaar ziet wat gerelateerd is aan de test-frameworks, mocht je ooit kijken naar de lijst van imports.

Reply:
1. Ok, ook niet erg als het zo blijft. 
2. Ja dat is prima, maar als we specificeren dat hij ook JSON, YAML, XML moet formatten, gaat onze IDE dat toch doen zodra je iets wijzigt in zo'n file? Zo niet, dan is het prima. Bij Java heb ik iig de ervaring dat hij het gelijk doet bij een save (of commit). 
3. Er is een Eclipse plugin voor zag ik, maar ik weet niet wat de actuele stand van zaken is. Misschien ook niet de moeite om daarop te richten. Ik bedenk me nu wel dat als we dit toepassen, onze Eclipse-gebruiker dan ook de import-order niet oppakt waardoor hij het nog steeds gaat herorderenen... dus dan schieten we niet veel op. Hij moet dan over, voordat we dit gaan uitvoeren op de gehele code base. 
4. Ja klopt hoor. Soms is een beetje breder prima, maar bv voor streams wil ik het graag ook zo houden dat hij het opsplitst over de regels. 

Mij betreft doen we niet veel uitzonderingen op die imports, behalve als het sterke meerwaarde heeft. Als alles netjes gesorteerd is, gaat het alweer een stuk beter. Als jij JUnit/Mockito nog wil toevoegen is dat voor mij op zich wel prima, maar ook goed om het zo te laten.

Reply:
Ik denk dat ik dit zo goed vindt, als ik kijk naar de andere PR, waarin het effect staat. De uitlijning met extra spaties (of tabs?) etc mag er van mij wel uit, zoals vanmorgen besproken, maar misschien zit dat al goed in deze PR.

Soms zie ik dat er 2 lege regels in de imports staan, wat ik vreemd vindt. Maar misschien is dat een logisch gevolg van de regels, zoals wij die nu hebben bedacht. Voorbeeld zie je bv bovenin bestand  `ExchangeMailListenerTestBase.java`.

Reply:
2 regels in imports had ik bewust gedaan om test-imports te scheiden van niet-test-imports. Maar dat kan ik er ook snel en makkelijk weer uithalen. 😃